src/test/java/uk/gov/ons/census/action/schedule/ActionRuleProcessorTest.java

```diff
@@ -55,10 +48,9 @@ private ActionRule setUpActionRule(ActionType actionType) {
     actionRule.setTriggerDateTime(OffsetDateTime.now());
     actionRule.setHasTriggered(false);
 
-    Map<String, List<String>> classifiers = new HashMap<>();
-    classifiers.put("A Key", new ArrayList<>());
+    String userDefinedWhereClause = " AND treatment_code IN ('abc', 'xyz')";
```

Comment:
This one still starts with `AND`, I'm wondering how that is passing since it should end up with `AND AND` in the resulting SQL?

Reply:
The actionscheduler " AND" in a test shouldn't be there.  In that test though anything will work, it's part of my dislike of some of these unit tests - actually testing the DB interaction in ITs feels more sensible to me.